src/main/java/org/opensearch/security/dlic/rest/api/ActionGroupsApiAction.java

```diff
@@ -88,8 +88,8 @@ protected CType getConfigName() {
 	}
 
 	@Override
-    protected String getResourceName() {
-        return "actiongroup";
+    	protected String getResourceName() {
```

Comment:
this change is outside of scope for this PR.

Reply:
The same applies to several other files.

Reply:
fixed